frontend/src/components/eMIB/EmibTabs.jsx

```diff
@@ -12,19 +12,16 @@ class EmibTabs extends Component {
       {
         id: 0,
         tabName: LOCALIZE.emibTest.tabs.instructionsTabTitle,
-        selected: false,
         body: <HowTo />
       },
       {
         id: 1,
         tabName: LOCALIZE.emibTest.tabs.backgroundTabTitle,
-        selected: true,
```

Comment:
does this mean we can no longer set the default tab you have selected?

Reply:
It is set in another parameter. Previously each tab had a selected t/f attribute AND there was a currentTab prop in TabNavigation, which was excessive, error prone, etc. This was a clean up of that